packages/connect-core/src/utils/transactions.ts

```diff
@@ -1,129 +1,76 @@
+import { Address } from '@aragon/connect-types'
 import {
   Contract,
   providers as ethersProviders,
   utils as ethersUtils,
 } from 'ethers'
+
 import { erc20ABI, forwarderAbi, forwarderFeeAbi } from './abis'
-import { isFullMethodSignature } from './app'
-import { FunctionFragment } from '../types'
+import { TokenData } from '../types'
 import App from '../entities/App'
-
-export interface Transaction {
-  data: string
-  from?: string
-  to: string
-}
-
-export interface TransactionWithTokenData extends Transaction {
-  token: {
-    address: string
-    value: string
-    spender: string
-  }
-}
+import Transaction from '../entities/Transaction'
 
 export async function createDirectTransaction(
-  sender: string,
-  destination: string,
-  methodAbiFragment: FunctionFragment,
+  sender: Address,
+  destination: Address,
+  methodAbiFragment: ethersUtils.FunctionFragment,
   params: any[]
 ): Promise<Transaction> {
-  let transactionOptions = {}
-
-  // If an extra parameter has been provided, it is the transaction options if it is an object
```

Comment:
To provide overrides is no longer relevant on the context of Connect as we just return a raw transaction object.